bosnet/sebak
# Enrich long-term test

### The new long-term test scenario
1. Run 4 nodes
1. Generate 10 accounts and check all balances
1. Payment from 10 accounts each other and check all balances
1. Kill and re-run all four nodes in order(sync test)
1. Payment from 10 accounts each other and check all balances
1. Check common accounts(inflation and fee)
1. Check total balance is 500,000,000 + inflation

fixes #674, #619 

Please check commit messages.
Almost every changes are for test case.
